omf-python/tests/test_texture_attribute.py

```diff
@@ -0,0 +1,42 @@
+import omf_python
+from os import path
+from unittest import TestCase
+
+
+class TestPointSet(TestCase):
```

Comment:
Maybe name this test `TestMappedTextureAttribute` and the file test_mapped_texture_attribute.py